zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/ErrorMapperTest.java

```diff
@@ -63,6 +64,8 @@ public class ErrorMapperTest {
   private static final String USER_TASKS_BASE_URL = "/v1/user-tasks";
 
   @MockBean UserTaskServices userTaskServices;
+  @MockBean GroupService groupService;
+  @MockBean BrokerClient brokerClient;
```

Comment:
🔧 For consistency, you could add them to the test application defined at the bottom of the class. The user task services are mocked up here because we're interacting with them.